### PR TITLE
Add shutdown flag for loop services

### PR DESCRIPTION
**Pull request by cicdw**
## Summary

Unused, intended for downstream use cases.

## Checklist

This PR:

- [x] adds new tests (if appropriate)
- [x] adds a change file in the `changes/` directory (if appropriate)

### CODE REVIEW

**Review by jlowin — APPROVED** [review]

File: src/prefect_server/api/runs.py [hunk 1]
```diff
@@ -334,7 +334,7 @@ async def get_or_create_task_run_info(
         id=run.returning.id,
         version=db_task_run.version,
         state="Pending",
-        serialized_state=db_task_run.serialized_state,
+        serialized_state=db_task_run_state.serialized_state,
```
Comment (madkinsz) [inline src/prefect_server/api/runs.py:337]: Find this mistake too?

File: src/prefect_server/services/loop_service.py [hunk 2]
```diff
@@ -19,6 +19,9 @@ class LoopService:
     # if no loop_seconds_config_key is provided, this will be the default
     loop_seconds_default = 600
 
+    # shutdown flag for gracefully exiting the infinite loop
+    is_running = True
```
Comment (madkinsz) [inline src/prefect_server/services/loop_service.py:23]: Does it make sense to have this be at the class level?

e.g.

```
In [1]: class foo:
   ...:     x = True
   ...: 

In [2]: foo.x
Out[2]: True

In [3]: instance = foo()

In [4]: instance.x
Out[4]: True

In [5]: foo.x = False

In [6]: instance.x
Out[6]: False
```

File: src/prefect_server/api/runs.py (hunk 1 above)
Comment (cicdw) [reply src/prefect_server/api/runs.py:337]: Yup, fixes an issue - I'll rebuild the relevant tagged server images after this is merged.

File: src/prefect_server/services/loop_service.py (hunk 2 above)
Comment (cicdw) [reply src/prefect_server/services/loop_service.py:23]: Great question - all services which run in the same process will shutdown simultaneously either way (there's no need to selectively shut down a single service), so it's not necessary to make it an instance variable.

**Review by madkinsz — APPROVED** [review]

File: src/prefect_server/services/loop_service.py (hunk 2 above)
Comment (madkinsz) [reply src/prefect_server/services/loop_service.py:23]: Might be worth noting in a comment but that makes sense!

**Pull request merged**